WebDriverAgentLib/Routing/FBSession.m

```diff
@@ -166,7 +166,16 @@ - (void)kill
 
 - (XCUIApplication *)activeApplication
 {
-  if (nil != self.testedApplication) {
+  BOOL isAuto = [self.defaultActiveApplication isEqualToString:FBDefaultApplicationAuto];
+  NSString *defaultBundleId = isAuto
+    ? nil
+    : self.defaultActiveApplication;
+
+  if (nil != defaultBundleId && [self applicationStateWithBundleId:defaultBundleId] >= XCUIApplicationStateRunningForeground) {
+    return [self makeApplicationWithBundleId:defaultBundleId];
+  }
+
+  if (isAuto && nil != self.testedApplication) {
```

Comment:
not sure if we still need to check isAuto flag here

Reply:
yea, i have tested a couple of scenarios to move applications